### ISSUE TITLE
file in use during installation

### ISSUE THREAD

**Issue by burmancomp** [nvaccess/nvda#16199]
Maybe pull request #16174 has something to do with this.
### Steps to reproduce:
- build launcher from source (nvda running)
- run new installer
- select to install to this computer
- when updating, "file in use" window appears (retry/cancel); because retry does not work installation fails.
### Actual behavior:

Installation (updating) fails.
### NVDA logs, crash dumps and other attachments:
[loglines.log](https://github.com/nvaccess/nvda/files/14345595/loglines.log)

#### NVDA installed/portable/running from source:
compiled from source
#### Windows version:
windows 10 22h2
#### Does the issue still occur after restarting your computer?
yes

**Comment by burmancomp** [comment]
I also rebooted so that nvda did not start, and then I started installer using narrator. Result seems to be same. If I select in "file in use" window to retry, result seems to be infinite loop where "file in use" window appears time after time.

Something has happened during installation retries because installer announces that I am about to replace newer version with older version.

**Comment by lukaszgo1** [comment]
@burmancomp  What version of NVDA you have installed currently? Is your Windows account an admin, and have you answered the UAC prompt after executing installation? Also, since you can run from sources it would be useful to know if NVDA can install successfully for you with #16174 reverted. For what its worth I cannot reproduce the issue on my machine, so there is an environmental part to this. For reference, the same error was posted in latest comment to #16122

**Comment by burmancomp** [comment]
Previous installations in that system have not suffered from this problem. I install quite often new version from current source in that system; I have added some braille related things but I have not changed them now thus they should not cause this. During installation I answer yes to uac question.

**Comment by burmancomp** [comment]
I reverted #16174 and installer worked normally.

**Comment by michaelDCurran** [comment]
Confirmed. On most machines, the installer is too quick after the UAC screen to try deleting the first exe. NvDA still takes a little bit of time to exit on the UAC screen.
Pr #16174 did however ensure that a user's installation is no longer left in a corrupt state if they cancel the install at this point which is most definitely an improvement. 
But obviously an installer that will never install is not sufficient. 
Therefore I have rolled back auto update to beta7 (before this got merged).
And I have pr #16205  to address this issue by again trying multiple times with a delay before completely failing.
